### ISSUE TITLE
[css-content] Initial value of quotes should be auto

### ISSUE THREAD

**Issue by emilio** [w3c/csswg-drafts#4074]
Right now https://drafts.csswg.org/css-content/#propdef-quotes says:

> Initial: depends on user agent 

WebKit and Blink have a magic initial value which you can only set with `initial` and which serializes to the empty string (which doesn't round-trip) and which triggers language-dependent quotes.

In Gecko we're interested in adopting a similar mechanism, but it'd be unfortunate if the initial value couldn't be represented in CSS (I think the fact that `getComputedStyle` returns the empty string is a bug).

Would it make sense to add an `auto` value which triggers this behavior and which is also the initial value?

See also: https://bugzilla.mozilla.org/show_bug.cgi?id=1421938

cc @jfkthame @fantasai @jensimmons

**Comment by xfq** [comment]
https://drafts.csswg.org/css-content-3/#quotes-property

The initial value of `quotes` is still "depends on user agent". Should it be `auto`?

**Comment by domenic** [comment]
It seems the intention was also to update https://html.spec.whatwg.org/#quotes, right?

**Comment by fantasai** [comment]
@xfq  Yes. :/ I noted it in Changes, but didn't fix the propdef... Fixed now, and republished.

@domenic Yes.

**Comment by domenic** [comment]
Is this just a matter of deleting all of https://html.spec.whatwg.org/#quotes now? I'm happy to post a PR for that if someone can confirm.

**Comment by lilles** [comment]
> Is this just a matter of deleting all of https://html.spec.whatwg.org/#quotes now? I'm happy to post a PR for that if someone can confirm.

That sounds correct to me.
